tools/azure-sdk-tools/packaging_tools/templates/MANIFEST.in

```diff
@@ -1,4 +1,4 @@
-recursive-include tests *.py *.yaml
+recursive-include tests *.py
```

Comment:
You don't own this file, that's a discussion to have outside of this PR with Zim.